js/renovation/ui/data_grid/data_grid.tsx

```diff
@@ -421,6 +431,11 @@ export class DataGrid extends JSXComponent(DataGridProps) {
     this.prevProps = this.props;
   }
 
+  @Effect()
+  dispose() {
```

Comment:
Add run: 'once' parameter to Effect. Or dispose will be called before real dispose.